users/app.py

```diff
@@ -60,5 +54,43 @@ def add_update_user():
     return jsonify(user_object), (201 if added_new_user else 200)
 
 
+def find_authorization_in_db(username, users_collection):
+    """Queries the db to find authorization of the given user.
+
+    Documents in the users collection should look like
+        {"username": "cmei4444",
+        "name": "Carolyn Mei",
+        "is_organizer": True}
+    """
+    if users_collection.count_documents({"username": username}) == 0:
+        return False  # user not found
+    cursor = users_collection.find({"username": username})
+    first_user = cursor[0]
+    authorized = first_user.get("is_organizer")
+    return False if authorized is None else authorized
```

Comment:
I think you can compare with `True` or use the `bool` constructor to convert `None` to `False`:

```python
return bool(user.get("is_organizer"))
```

Reply:
Fixed https://github.com/knative-portability/large-events/pull/2/commits/a2adc51ddc490f7dbca20b6abe1833efe75bb784